js-v1/src/rpc.ts

```diff
@@ -12,9 +12,9 @@ export async function getStakeActivation(
   connection: Connection,
   stakeAddress: PublicKey
 ): Promise<StakeActivation> {
-  const SYSVAR_STAKE_HISTORY_ADDRESS = new PublicKey(
-    'SysvarStakeHistory1111111111111111111111111'
-  );
+  const SYSVAR_STAKE_HISTORY_ADDRESS = {
+    toBase58: () => 'SysvarStakeHistory1111111111111111111111111',
```

Comment:
This works, because the only thing that `parseAccountInfo()` ever does with this thing is to call `toBase58()` on it.

‘Thin interfaces.’